student.rb

```diff
@@ -1,6 +1,8 @@
 class Student < Person
+  attr_accessor :classroom
+
   def initialize(classroom, age, name: 'unknown', parent_permission: true)
-    super(age, name, parent_permission,)
+    super(age, name, parent_permission)
     @classroom = classroom
   end
```

Comment:
- [ ] there should be a method that serves as the 'belongs-to' side (a student belongs to a classroom). have a look at [the article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships) with examples